scripts/dbConfig.py

```diff
@@ -1,61 +1,9 @@
 
 
 dbConfig = {
-            'user': '',
-            'password': '',
-            'host': '',
-            'db': ''
+            'user': 'root',
+            'password': 'Logoslogos88',
```

Comment:
I don't think you meant to push your password here!

I'd suggest pulling these values from the .env file (using something like [python-dotenv](https://pypi.org/project/python-dotenv/) ). That way, you only ever have 1 place to keep updated with these confidential config variables.